rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -51,9 +51,9 @@ s32 npDrmIsAvailable(u32 k_licensee_addr, vm::cptr<char> drm_path)
 
 	if (k_licensee_addr)
 	{
-		for (s32 i = 0; i < 0x10; i++)
+		for (s8 i = 0; i < 0x10; i++)
 		{
-			k_licensee[i] = vm::read8(k_licensee_addr + i);
+			k_licensee[i] = *k_licensee_addr + i;
```

Comment:
Wouldn't it behave differently, from `vm::read8(k_licensee_addr + i);`?
Shouldn't it be `*(k_licensee_addr + i);`?